src/client/containers/FavoritePage/FavoritePage.Container.js

```diff
@@ -0,0 +1,61 @@
+import './FavoritePage.styles.css';
+import React, { useState, useEffect, useCallback } from 'react';
+import FavoriteList from '../../components/FavoriteList/FavoriteList.component';
+
+export const addProductToFavorites = (userId, productId) => {
+  console.log('in add favorites container');
+
+  const requestOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      product_id: productId,
+      user_id: userId,
+    }),
+  };
+  fetch('/api/favorites/add', requestOptions)
+    .then((response) => response.json())
+    .then((result) => {
+      console.log(result);
+    });
+};
+
+export default function FavoritePage() {
+  const [favorites, setFavorites] = useState([]);
+
+  const GetFavorites = useCallback(() => {
+    const apiUrl = '/api/users/10/favorites';
```

Comment:
It is hardcoded to fetch favorites of user with id 10! it should be changed!

Reply:
this is done for testing only!should be updated after user authentication

Reply:
How about putting a `TODO` comment here?